### PR TITLE
cats: add Teawie-Prison

### PR DESCRIPTION
**Pull request by Scrumplex**
Out first official Cat Pack!

### CODE REVIEW

File: cats/Teawie-Prison/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Teawie Pack, but for some reason he was imprisoned
+
+## Preview
+![Preview](preview.png)
+
+## License
+```
+SPDX-FileCopyrightText: 2024 Sefa Eyeoglu <contact@scrumplex.net>
+
+SPDX-License-Identifier: MIT
```
Comment (DioEgizio) [inline cats/Teawie-Prison/README.md:18]: If I understand correctly, the json is MIT while the actual art is cc by-sa. Shouldn't it say both?

Comment (Scrumplex) [reply cats/Teawie-Prison/README.md:18]: I guess we would ideally add all licenses and copyright holders, I agree. Lemme update the README template too then

Comment (Scrumplex) [reply cats/Teawie-Prison/README.md:20]: done

File: cats/Teawie-Prison/preview.png.license [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2024 Sefa Eyeoglu <contact@scrumplex.net?
```
Comment (TheKodeToad) [inline cats/Teawie-Prison/preview.png.license:1]: scrumplex?

Comment (Scrumplex) [reply cats/Teawie-Prison/preview.png.license:1]: lol

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**